packages/frontend-2/components/viewer/resources/ModelCard.vue

```diff
@@ -109,6 +123,7 @@ import {
   useInjectedViewerRequestedResources
 } from '~~/lib/viewer/composables/setup'
 import { useDiffUtilities } from '~~/lib/viewer/composables/ui'
+import ActiveVersionCard from './ActiveVersionCard.vue'
```

Comment:
1. always use absolute path imports
2. don't import local components, nuxt has auto-imports for those